proxy/src/main/java/com/wavefront/agent/queueing/TaskQueueFactoryImpl.java

```diff
@@ -110,14 +111,31 @@ public Long value() {
     try {
       File lockFile = new File(lockFileName);
       if (lockFile.exists()) {
-        Preconditions.checkArgument(true, lockFile.delete());
+        if (!lockFile.delete()) {
```

Comment:
Use `Files.Delete()` it give more info a better exceptions.
https://docs.oracle.com/javase/tutorial/essential/io/delete.html

Reply:
or maybe best to use [deleteIfExists(Path)](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#deleteIfExists-java.nio.file.Path-)